<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_product</class>
 <widget class="QWidget" name="page_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #58595B;
font-family: Montserrat;
font-style: light;
font-weight: normal;
font-size: 23px;
line-height: 29px;</string>
  </property>
  <widget class="QPushButton" name="pushButton_previous_page">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>1757</y>
     <width>281</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_continue">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>1541</y>
     <width>740</width>
     <height>92</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_to_help">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>1770</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_price_custom">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>1160</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
background-image: url(background.png);
font-style: normal;
font-weight: bold;
font-size: 36px;
line-height: 44px;
color: #FFFFFF;
text-align: center;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_price_large">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>1040</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
background-image: url(/home/df-admin/production/references/background.png);
font-style: normal;
font-weight: bold;
font-size: 36px;
line-height: 44px;
color: #3D6675;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_size_custom">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>1110</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #5E8680;
font-family: Montserrat;
font-style: light;
font-weight: normal;
font-size: 20px;
line-height: 24px;
text-align: center;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_size_large">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>1000</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #5E8680;
font-family: Montserrat;
font-style: light;
font-weight: normal;
font-size: 20px;
line-height: 24px;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_order_small">
   <property name="geometry">
    <rect>
     <x>564</x>
     <y>996</y>
     <width>130</width>
     <height>100</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_order_big">
   <property name="geometry">
    <rect>
     <x>858</x>
     <y>996</y>
     <width>130</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_order_medium">
   <property name="geometry">
    <rect>
     <x>711</x>
     <y>996</y>
     <width>130</width>
     <height>100</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_order_custom">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>1113</y>
     <width>424</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_size_medium">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>1000</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #5E8680;
font-family: Montserrat;
font-style: light;
font-weight: normal;
font-size: 20px;
line-height: 24px;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_price_medium">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>1040</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
background-image: url(/home/df-admin/production/references/background.png);
font-style: normal;
font-weight: bold;
font-size: 36px;
line-height: 44px;
color: #3D6675;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_size_small">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>1000</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #5E8680;
font-family: Montserrat;
font-style: light;
font-weight: normal;
font-size: 20px;
line-height: 24px;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_price_small">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>1040</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
background-image: url(/home/df-admin/production/references/background.png);
font-style: normal;
font-weight: bold;
font-size: 36px;
line-height: 44px;
color: #3D6675;
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_title">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>326</y>
     <width>881</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_description">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>461</y>
     <width>901</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_ingredients">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1280</y>
     <width>888</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_photo">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>760</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string>picture</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_ingredients_title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1239</y>
     <width>351</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingredients:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_background_small">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background_medium">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background_large">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_background_custom">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_select_quantity">
   <property name="geometry">
    <rect>
     <x>567</x>
     <y>949</y>
     <width>393</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_help">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>1773</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_help</zorder>
  <zorder>pushButton_continue</zorder>
  <zorder>pushButton_order_small</zorder>
  <zorder>pushButton_order_big</zorder>
  <zorder>pushButton_order_medium</zorder>
  <zorder>pushButton_order_custom</zorder>
  <zorder>label_size_medium</zorder>
  <zorder>label_price_medium</zorder>
  <zorder>label_size_small</zorder>
  <zorder>label_price_small</zorder>
  <zorder>label_price_custom</zorder>
  <zorder>label_size_custom</zorder>
  <zorder>label_price_large</zorder>
  <zorder>label_size_large</zorder>
  <zorder>label_product_title</zorder>
  <zorder>label_product_description</zorder>
  <zorder>label_product_photo</zorder>
  <zorder>label_product_ingredients_title</zorder>
  <zorder>label_background_small</zorder>
  <zorder>label_background_medium</zorder>
  <zorder>label_background_large</zorder>
  <zorder>label_background_custom</zorder>
  <zorder>label_product_ingredients</zorder>
  <zorder>label_select_quantity</zorder>
  <zorder>pushButton_back</zorder>
  <zorder>pushButton_to_help</zorder>
  <zorder>pushButton_previous_page</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
